caffeine/src/test/java/com/github/benmanes/caffeine/cache/testing/CacheGenerator.java

```diff
@@ -217,7 +217,7 @@ private static <K, V> Cache<K, V> newCache(CacheContext context) {
   @SuppressWarnings("unchecked")
   private static void populate(CacheContext context, Cache<Int, Int> cache) {
     if (context.population.size() == 0) {
-      // timeWhel clock initialization
+      // timeWheel clock initialization
```

Comment:
??

Reply:
meant to be `timerWheel`. This is a data structure (see TimerWheel.java) also referred to as a timing wheel. It is an O(1) priority queue based on hashing.